<dec f='boost-1.85.0/libs/test/include/boost/test/test_framework_init_observer.hpp' l='42' type='void boost::unit_test::framework_init_observer_t::exception_caught(const boost::execution_exception &amp; )'/>
<inh f='boost-1.85.0/libs/test/include/boost/test/tree/observer.hpp' l='98' c='_ZN5boost9unit_test13test_observer16exception_caughtERKNS_19execution_exceptionE'/>
<def f='boost-1.85.0/libs/test/include/boost/test/impl/test_framework_init_observer.ipp' l='60' ll='64' type='void boost::unit_test::framework_init_observer_t::exception_caught(const boost::execution_exception &amp; )'/>
<doc f='boost-1.85.0/libs/test/include/boost/test/impl/test_framework_init_observer.ipp' l='58'>//____________________________________________________________________________//</doc>
